From c64f7a81c5842bc7fb7f68060317139555bb4cbf Mon Sep 17 00:00:00 2001
From: Bogdan-Gabriel Roman <bogdan-gabriel.roman@nxp.com>
Date: Tue, 26 Oct 2021 16:30:35 +0300
Subject: [PATCH 16/16] dt-bindings: s32gen1: remove unused clock frequencies

S32GEN1_CLK_ARM_PLL_DFS1_FREQ is not used anymore.
S32GEN1_XBAR_2X_FREQ is platform specific and its definition is used
only in platform specific clock files as S32GEN1_XBAR_2X_MAX_FREQ right
now.

Issue: ALB-7717
Upstream-Status: Pending 

Signed-off-by: Bogdan-Gabriel Roman <bogdan-gabriel.roman@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 include/dt-bindings/clock/s32gen1-clock-freq.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/include/dt-bindings/clock/s32gen1-clock-freq.h b/include/dt-bindings/clock/s32gen1-clock-freq.h
index 6fe3a1fdc..094711abe 100644
--- a/include/dt-bindings/clock/s32gen1-clock-freq.h
+++ b/include/dt-bindings/clock/s32gen1-clock-freq.h
@@ -23,8 +23,6 @@
 #define S32GEN1_ARM_PLL_PHI0_MAX_FREQ		(1000 * MHZ)
 #define S32GEN1_PERIPH_PLL_PHI0_MIN_FREQ	(100 * MHZ)
 #define S32GEN1_PERIPH_PLL_PHI2_MIN_FREQ	(40 * MHZ)
-#define S32GEN1_CLK_ARM_PLL_DFS1_FREQ		(800 * MHZ)
-#define S32GEN1_XBAR_2X_FREQ			(800 * MHZ)
 
 #elif defined(PLAT_s32g3)
 #define S32GEN1_A53_MAX_FREQ			(1300 * MHZ)
@@ -32,8 +30,6 @@
 #define S32GEN1_ARM_PLL_PHI0_MAX_FREQ		(1300 * MHZ)
 #define S32GEN1_PERIPH_PLL_PHI0_MIN_FREQ	(100 * MHZ)
 #define S32GEN1_PERIPH_PLL_PHI2_MIN_FREQ	(40 * MHZ)
-#define S32GEN1_CLK_ARM_PLL_DFS1_FREQ		(793236859UL)
-#define S32GEN1_XBAR_2X_FREQ			(793236859UL)
 #else
 #error Undefined platform
 #endif
-- 
2.17.1

